content/en/docs/hrt/singapore/singapore-hrt-overview.md

```diff
@@ -7,17 +7,17 @@ weight: 1
 
 ## The Process
 
-The main place to get HRT is at [Changi General Hospital](https://www.cgh.com.sg/). Your first step depends on whether you are a singaporean as it determines if you are eligible for goverment subsidies.
+The main place to get HRT is [Changi General Hospital](https://www.cgh.com.sg/). Your first step depends on whether you are a singaporean as it determines if you are eligible for government subsidies.
 
-1. Polyclinic referral for subsidies *(non-singaporeans skip this step)*
+1. Polyclinic referral for subsidies *(non-singaporeans may skip this step)*
```

Comment:
I think should not add 'may' because non-singaporeans are certainly not possible to get the referral for subsidies